egs/talromur/voc1/conf/slurm.conf

```
@@ -0,0 +1,12 @@
+# Default configuration
+command sbatch --export=PATH  --ntasks-per-node=1
+option time=* --time $0
+option mem=* --mem-per-cpu $0
+option mem=0          # Do not add anything to qsub_opts
+option num_threads=* --cpus-per-task $0 --ntasks-per-node=1
+option num_threads=1 --cpus-per-task 1  --ntasks-per-node=1 # Do not add anything to qsub_opts
+default gpu=0
+option gpu=0 -p shared
+option gpu=* -p longrunning --gres=gpu:$0 --time=14-00:00
```

Changed to default setting.
```suggestion
option gpu=0 -p cpu
option gpu=* -p gpu --gres=gpu:$0
```